extensions/wikia/AdEngine/js/slot/floatingMedrec.js

```diff
@@ -57,21 +57,21 @@ define('ext.wikia.adEngine.slot.floatingMedrec', [
 				});
 			}
 
-			if ($win.scrollTop() > startPosition && $win.scrollTop() < stopPosition) {
-				$adSlot.css({
-					position: 'fixed',
-					top: globalNavigationHeight + margin + 'px',
-					visibility: 'visible'
-				});
-			}
-
-			if ($win.scrollTop() >= stopPosition) {
-				$adSlot.css({
-					position: 'absolute',
-					top: stopPosition - startPosition + 'px',
-					visibility: 'visible'
-				});
-			}
+			// if ($win.scrollTop() > startPosition && $win.scrollTop() < stopPosition) {
```

Comment:
If we use AB test condition in the [Rail.js|https://github.com/Wikia/app/blob/dev/extensions/wikia/Rail/scripts/Rail.js#L82] it should be safe to merge it and push live next week :)